### ISSUE TITLE
Support Flux.1

### ISSUE THREAD

**Issue by MohamedAliRashad** [damian0815/compel#99]
Can Flux.1 be supported ?

**Comment by damian0815** [comment]
sure if you want to buy me a gpu that can hold 72B params

**Comment by RedAndr** [comment]
> Can Flux.1 be supported?

Better switch to https://github.com/xhinker/sd_embed
It isn't that flexible and convenient but supports SD3 and FLUX.1.

**Comment by RedAndr** [comment]
@bonlime Agree. I wouldn't say FLUX.1 generates worse results with sd_embed, but it definitely quite different. It should be a bug. Compel at least gives exactly the same images. But unfortunately, there is not choice.

**Comment by mantrakp04** [comment]
> sure if you want to buy me a gpu that can hold 12B params

Hmm, how much would it cost approx. I can pitch in 100 bucks.

**Comment by hervenivon** [comment]
Happy to support here 👋

**Comment by damian0815** [comment]
Flux support has been added with v2.2.0
